ansys/dpf/core/dpf_operator.py

```diff
@@ -223,15 +225,15 @@ def inputs(self):
     
     @property
     def outputs(self):
-        """Enables to get outputs of the operator by evaluationg it
+        """Retrieve outputs of the operator by evaluating it.
```

Comment:
@PipKat operator.outputs doesn't evaluate an operator, it just gives access to available outputs. For example u.outputs doesn't evaluate but u.outputs.fields_container() does. I don't know how we can explain that